### PR TITLE
[connector-v1][doc]Fix inconsistency between document and code logic

### PR DESCRIPTION
**Pull request by laglangyue**
## Purpose of this pull request

#2742

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Comment by laglangyue** [comment]
The description of the document is a little vague, which makes it easy for users to step on it

**Comment by laglangyue** [comment]
close https://github.com/apache/incubator-seatunnel/issues/2742

File: docs/en/connector/sink/Jdbc.mdx [hunk 1]
```diff
@@ -56,7 +56,7 @@ user password
 
 ### dbTable [string]
 
-Source data table name
+sink source table name, which will be created if not existed
```
Comment (TyrantLucifer) [inline docs/en/connector/sink/Jdbc.mdx]: Sink table name, not sink source table name, this is ambiguous. The best description is that `Sink table name, if this table is not existed, it wiil be created`.

File: docs/en/connector/sink/Jdbc.mdx [hunk 2]
```diff
@@ -75,9 +75,9 @@ The transaction isolation level, which applies to current connection. The defaul
 ### customUpdateStmt [string]
 
 Configure when `saveMode` is specified as `update` , which is used to specify the update statement template for key conflicts
-
-Refer to the usage of `INSERT INTO table (...) values (...) ON DUPLICATE KEY UPDATE... ` of `mysql` , use placeholders or fixed values in `values`
-
+if `customUpdateStmt` is empty ,the sql will auto-generate for all columns, else use your sql which refer to the usage of
+`INSERT INTO table (...) values (...) ON DUPLICATE KEY UPDATE... ` of `mysql` , use placeholders or fixed values in `values`
+tips: the tableName of sql should be consistent with the dbName.
```
Comment (TyrantLucifer) [inline docs/en/connector/sink/Jdbc.mdx]: What's the meaning of this? Do you mean that the table in sql statement should be consistent with `dbTable`? If so, please describe it more clearly

Comment (laglangyue) [reply docs/en/connector/sink/Jdbc.mdx]: image

1. In many scenarios, do not need to customize the updateSQL, which can be generated automatically. In addition, user-defined sql is prone to exceptions that the number of parameters does not match.

2. If need customize SQL exactly, the table name of SQL should be consistent with that of dbTable

Comment (TyrantLucifer) [reply docs/en/connector/sink/Jdbc.mdx]: Yep, but in your description. it's `dbName`

image

File: docs/en/connector/sink/Jdbc.mdx (hunk 1 above)
Comment (laglangyue) [reply docs/en/connector/sink/Jdbc.mdx]: done

File: docs/en/connector/sink/Jdbc.mdx (hunk 2 above)
Comment (laglangyue) [reply docs/en/connector/sink/Jdbc.mdx]: My mistake,I fix it.

**Review by TyrantLucifer — APPROVED** [review]

**Review by EricJoy2048 — APPROVED** [review]
+1

**Pull request merged**